lib/linter.js

```diff
@@ -0,0 +1,29 @@
+const _ = require('lodash');
+const Result = require('./result-model');
+
+class Linter {
+  constructor(options) {
+    this.matcher = options.matcher;
+    this.pathParser = options.pathParser || _.identity;
+
+    _.bindAll(this, ['filePathTest', 'lint']);
```

Comment:
So that these methods can be passed in directly where their needed without using `.bind(context)`